egs2/slue-voxpopuli/asr1/local/score.py

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/env bash
```

Comment:
I believe you already have some checks, but is that possible that we can reuse some existing scoring for slu?

Reply:
Actually, this scoring is entity classification task whereas check we have is for measuring intent classification performance. Hence I can not reuse the existing scoring for slu.